specification/app/resource-manager/Microsoft.App/stable/2024-03-01/Diagnostics.json

```diff
@@ -432,6 +432,154 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/jobs/{jobName}/detectors": {
+      "get": {
+        "tags": [
+          "Jobs",
+          "Diagnostics"
+        ],
+        "summary": "Get the list of diagnostics for a given Container App Job.",
+        "description": "Get the list of diagnostics for a Container App Job.",
+        "operationId": "Jobs_ListDetectors",
```

Comment:
Does not conform with https://github.com/Azure/azure-openapi-validator/blob/main/docs/operation-id-noun-verb.md

Reply:
Discussed offline, this breaks existing SDK but since we are api-version is in preview and it is internal api. It is okay to change